File: Algorithms/flutter_algorithms/test/algorithms/ackermann_function_test.dart
import 'package:flutter_test/flutter_test.dart';
import 'package:flutter_algorithms/algorithms/ackermann_function.dart';

void main() {
  test('AckermannFunction returns valid value for small input', () {
    expect(AckermannFunction.call(2, 3), 9);
  });

  test('AckermannFunction returns valid value for big input', () {
    expect(AckermannFunction.call(3, 5), 253);
  });
}

File: Algorithms/ReactNativeAlgorithms/__tests__/algorithms/ackermannFunction-test.js
import ackermannFunction from '../../src/algorithms/ackermannFunction';

describe('ackermannFunction', () => {
  it('returns correct value for small input', () => {
    expect(ackermannFunction(2, 3)).toBe(9);
  });

  it('returns correct value for small medium input', () => {
    expect(ackermannFunction(3, 5)).toBe(253);
  });
});